mkdocs.yml

```diff
@@ -122,3 +127,15 @@ theme:
   features:
     - navigation.tabs
     - navigation.sections
+
```

Comment:
Have you added these extensions also in the dsp-docs repository? Otherwise, the local build will succeed, but on https://docs.dasch.swiss, it will fail.

Reply:
[yes](https://github.com/dasch-swiss/dsp-docs/commit/c929a2229362817b798eaf4013740ba616b61f23) :)